README.md

```diff
@@ -158,57 +160,9 @@ The `/static` and `/template` directories are replicated in `out/`, and the
 files in `src/` are rebased into `out/` (so `src/base/foo.py` becomes
 `out/base/foo.py`).
 
-
-## Detailed Dependency Information
--------------
-* The AppEngine SDK should be present in the directory:
```

Comment:
Some of this is still relevant, right?  e.g. the $HOME/bin/google_appengine/ for appcfg.py and so on I think are, as they are used in manageScript, etc. of the appengine npm dependency for deployment?